src/GitVersionExe/Program.cs

```diff
@@ -129,17 +130,27 @@ static void ConfigureLogging(Arguments arguments)
             {
                 try
                 {
-                    Directory.CreateDirectory(Path.GetDirectoryName(arguments.LogFilePath));
-                    if (File.Exists(arguments.LogFilePath))
+                    var logFileFullPath = Path.GetFullPath(arguments.LogFilePath);
+                    var logFile = new FileInfo(logFileFullPath);
+
+                    // NOTE: logFile.Directory will be null if the path is i.e. C:\logfile.log. @asbjornu
+                    if (logFile.Directory != null)
+                    {
+                        logFile.Directory.Create();
+                    }
+
+                    if (!logFile.Exists)
```

Comment:
@asbjornu we'll possible hit some regression due to this if-statement, since it means that we're appending the log instead of overwriting the file as we're doing in the current `master`